src/astro/sql/operators/dataframe.py

```diff
@@ -130,4 +134,10 @@ def execute(self, context: Dict) -> Union[Table, pd.DataFrame]:
             )
             return self.output_table
         else:
+            if (
+                conf.get("core", "xcom_backend") == "airflow.models.xcom.BaseXCom"
+                and os.getenv("ASTRO__DATAFRAME__ALLOW_UNSAFE_STORAGE", "False").lower()
```

Comment:
```suggestion
                and os.getenv("AIRFLOW__ASTRO_SDK__DATAFRAME_ALLOW_UNSAFE_STORAGE", "False").lower()
```

Reply:
use `conf.get` as this setting should come from there too like - https://github.com/astronomer/astro-sdk/pull/522#discussion_r917727385